content/blog/esc-projects-environment-tags-launch/index.md

```diff
@@ -63,6 +63,13 @@ We offer numerous ways for you to get started with Projects via the Pulumi Web C
   pulumi/cloud-pe/demo
  ```
 
+### Breaking Changes
```

Comment:
```suggestion
### OIDC Subject Changes
```

Reply:
The changes here are both for OIDC subjects as well as environment context

Reply:
Updating! Didn't want to call them "breaking changes" because it isn't.